backend/fms_core/viewsets.py

```diff
@@ -518,13 +597,11 @@ def summary(self, _request):
         for eg in Sample.objects.values_list("experimental_group", flat=True):
             experimental_groups.update(eg)
 
-        sample_kind_names_by_id = {sample_kind.id: sample_kind.name for sample_kind in SampleKind.objects.all()}
-
+        
         return Response({
             "total_count": Sample.objects.all().count(),
-            "extracted_count": SampleLineage.objects.all().count(),  # WARNING !!! will need a filter when transfer are added
             "kinds_counts": {
-                sample_kind_names_by_id[c["sample_kind"]]: c["sample_kind__count"]
+                c["sample_kind"]: c["sample_kind__count"]
```

Comment:
This means the sample kinds being displayed by id rather than by name?

Reply:
yeah this means the count is returned by id instead of name. Since i did the change on protocol i made the change for the kinds too so we would be coherent.